hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -251,6 +251,24 @@ objects:
         kind: ClusterRole
         name: splunk-forwarder-operator
 
+
+- apiVersion: hive.openshift.io/v1
+  kind: SelectorSyncSet
+  metadata:
+    name: splunk-forwarder-operator-osd-splunk-auth
+    namespace: splunk-forwarder-operator
+  spec:
+    clusterDeploymentSelector:
+      matchExpressions:
+      - { key: api.openshift.com/managed, operator: In, values: ["true"] }
+      - { key: api.openshift.com/environment, operator: In, values: ["production", "staging"] }
+      - { key: api.openshift.com/noalerts, operator: NotIn, values: ["true"] }
+      - { key: ext-managed.openshift.io/noalerts, operator: NotIn, values: ["true"] }
+      - key: api.openshift.com/legal-entity-id
+        operator: NotIn
+        values: ${{ALERTS_SKIP_LEGALENTITY_IDS}}
+      - { key: hive.openshift.io/cluster-region, operator: NotIn, values: ["cn-north-1", "cn-northwest-1"] }
+      - { key: api.openshift.com/fedramp, operator: DoesNotExist }
     secretMappings:
```

Comment:
Are secretMappings still needed?

Reply:
this targets OSD so i think so (although doesn't looks like **secretReferences** are a thing in hive any more)